notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<p>should we mention here that their presentation should be a 15 minute video with slides?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>